nni/experiment/nni_client.py

```diff
@@ -228,7 +228,7 @@ def __repr__(self):
                     .format(self.trialJobId, self.status, self.hyperParameters, self.logPath,
                             self.startTime, self.endTime, self.finalMetricData, self.stderrPath)
 
-class Experiment:
+class ExternalExperiment:
```

Comment:
What is external experiment?

Reply:
It means experiment created by other process.
You can suggest a better name.

Reply:
what scenario is it?